OpenRA.Mods.Common/UpdateRules/Rules/20180923/ChangeTakeOffSoundAndLandingSound.cs

```diff
@@ -0,0 +1,41 @@
+#region Copyright & License Information
+/*
+ * Copyright 2007-2018 The OpenRA Developers (see AUTHORS)
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation, either version 3 of
+ * the License, or (at your option) any later version. For more
+ * information, see COPYING.
+ */
+#endregion
+
+using System.Collections.Generic;
+
+namespace OpenRA.Mods.Common.UpdateRules.Rules
+{
+	public class ChangeTakeOffSoundAndLandingSound : UpdateRule
+	{
+		public override string Name { get { return "Change 'TakeOffSound' and 'LandingSound' parameters within 'Aircraft' Trait."; } }
+		public override string Description
+		{
+			get
+			{
+				return "The 'TakeOffSound' and 'LandingSound' parameters within 'Aircraft' have been changed\n" +
+					"to accept an array of playable sounds.\n" +
+					"They were renamed to 'TakeOffSounds' and 'LandingSounds' respectively, to reflect this change.\n" +
+					"Definitions of 'TakeOffSound' and 'LandingSound' will be automatically renamed.";
+			}
+		}
+
+		public override IEnumerable<string> UpdateActorNode(ModData modData, MiniYamlNode actorNode)
+		{
+			foreach (var tos in actorNode.ChildrenMatching("Aircraft"))
+				tos.RenameChildrenMatching("TakeOffSound", "TakeOffSounds");
+
+			foreach (var lds in actorNode.ChildrenMatching("Aircraft"))
```

Comment:
Please combine both those loops into one, like for example
```
foreach (var aircraft in actorNode.ChildrenMatching("Aircraft"))
{
	aircraft.RenameChildrenMatching("TakeOffSound", "TakeOffSounds");
	aircraft.RenameChildrenMatching("LandingSound", "LandingSounds");
}
```

Reply:
I tried doing that and it won't compile because the variable is already used once.
I don't know how else to write it?

Reply:
@Inq8: using @abcdefg30's code, I didn't get any compile errors. Please re-check your local code, or just replace it with the above.

Once that works, it would be great if you could squash/fixup all the update commits into the first, I think then the PR would be ready to merge.